web/settings.py

```diff
@@ -138,33 +138,39 @@ def RUNTIME_ENVIRONMENT():
 
 WSGI_APPLICATION = "web.wsgi.application"
 
-
 # Database
 # https://docs.djangoproject.com/en/5.1/ref/settings/#databases
 
+# In Azure, the Postgres server is running in a container app inside the same
+# container app environment as the web container app (Django)
+#
+# The Postgres container doesn't allow ingress from the public Internet, only
+# from within the container app environment, and all traffic stays within Azure
+# (i.e. doesn't travel over the public Internet)
+#
+# The Postgres container is also not setup for SSL connections right now
+PG_CONFIG = {
+    "ENGINE": "django.db.backends.postgresql",
+    "HOST": os.environ.get("POSTGRES_HOSTNAME", "postgres"),
+    "PORT": os.environ.get("POSTGRES_PORT", "5432"),
+    # https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
+    "OPTIONS": {"sslmode": "disable"},
+}
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.postgresql",
+    "default": PG_CONFIG
+    | {
```

Comment:
This is the Union operator for dicts in Python: https://peps.python.org/pep-0584/

Essentially the same as doing e.g.

```
settings = {}
settings.update(PG_CONFIG)
settings.update({"NAME": "...", ...})
```

But since `.update()` returns `None`, it can't be used inline like this.